keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -224,8 +224,7 @@ The complete expansion and recovery flow of both control-plane and kubelet is do
 ### Risks and Mitigations
 
 - Once expansion is initiated, the lowering of requested size is only allowed upto a value *greater* than `pvc.Status`. It is not possible to entirely go back to previously requested size. This should not be a problem however in-practice because user can retry expansion with slightly higher value than `pvc.Status` and still recover from previously failing expansion request.
-
-
+ 
 ## Graduation Criteria
 
 * *Alpha* in 1.23 behind `RecoverExpansionFailure` feature gate with set to a default of `false`.
```

Comment:
It would be nice if we can rollback all the way to the original size before before beta, because that would involve updating API validation. I need to think more deeply if we would need the new validation logic to soak for a release before going to beta.